frontend/apps/reader/modules/readerbookmark.lua

```diff
@@ -1012,6 +1014,25 @@ function ReaderBookmark:updateBookmark(item)
     end
 end
 
+
+function ReaderBookmark:patchBookmark(item)
+    for i=1, #self.bookmarks do
+        if item.datetime == self.bookmarks[i].datetime and item.page == self.bookmarks[i].page then
+            local bookmark_before = util.tableDeepCopy(self.bookmarks[i])
+            local is_auto_text = self:isBookmarkAutoText(self.bookmarks[i])
+            for key, val in pairs(item.updated_highlight) do
```

Comment:
Can you show please where `updated_highlight` is initialized. Cannot trace.

Reply:
My bad. Fixing it.

Reply:
Done.

Reply:
I updated keys where this function is being used.